lib/utils/getoptions.js

```diff
@@ -23,7 +23,9 @@ module.exports = function cwdResolver( callOptions, cwd ) {
 		resolverPath: path.resolve( __dirname, '../default-resolver.js' ),
 		resolverUrlTemplate: 'git@github.com:${ path }.git',
 		resolverTargetDirectory: 'git',
-		resolverDefaultBranch: 'master'
+		resolverDefaultBranch: 'master',
+		ignore: null,
+		scope: null
 	};
 
 	if ( fs.existsSync( mgitJsonPath ) ) {
```

Comment:
And with [L35](https://github.com/cksource/mgit2/pull/51/files#diff-da67e5c5e86dbd7f7f1844dcab3edeb0R35) as well.